high_card.rb

```diff
@@ -1,5 +1,65 @@
-# Use these two arrays to generate a deck of cards.
-ranks = ["A", 2, 3, 4, 5, 6, 7, 8, 9, 10, "J", "Q", "K" ]
-suits = [ "hearts", "spades", "clubs", "diamonds" ]
+def new_deck
+  ranks = [2, 3, 4, 5, 6, 7, 8, 9, 10, "J", "Q", "K", "A" ]
+  suits = [ "hearts", "spades", "clubs", "diamonds" ]
+  deck = []
+  suits.each do |suit|
+    ranks.each_with_index do |rank, value|
+      deck << [rank, value + 2, suit]
```

Comment:
could also use a hash instead of array here. will be more reliable in the future to access values